package test.exceptions;

public class ex1 {

	public static void main(String[] args) {
		Object o = Integer.valueOf(1532);
		String s = (String)o;
		System.out.println(o);
	}

}
